### ISSUE TITLE
[BUG] : Installation fails with Conda

### ISSUE THREAD

**Issue by andrewtarzia** [EISy-as-Py/hardy#9]
**Describe the bug**
Readme states Python 3.6 =>. Using a fresh conda environment with Python 3.6 leads to the following error upon installation:

```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - hardy -> python[version='>=3.7,<3.8.0a0']

Your python: python=3.6

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.

The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.31=0
  - feature:|@/linux-64::__glibc==2.31=0

Your installed version is: 2.31
```

I assume this is due to Python 3.6, but also note the issue with glibc, which I am unsure about.
Tensorflow is installed via `pip install tensorflow`.
Tested with Python 3.9, same issue.
Tested with Python 3.7, works, although I will open a new issue about running things in this environment.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

Apologies if I am missing something in the installation steps to get this to work.

**Comment by amoeez** [comment]
@andrewtarzia We started having troubles with Conda install on python 3.6 couple of months back (appeared in Travis CI). We traced changes in packages version upgrades, however, it didn't produce any meaningful solution. The only option we currently have is to the keep the python version restricted to 3.7 only.
I hope this will help. The change in dependency requirement has been added to the Readme and the documentations.
